functests/performance/updating_profile.go

```diff
@@ -133,6 +133,32 @@ var _ = Describe("[rfe_id:28761] Updating parameters in performance profile", fu
 				}
 			}
 		})
+
+		It("[test_id:22764] verify that realTimeKernel is default when realTimeKernel field does not exist", func() {
+			conditionType := machineconfigv1.MachineConfigPoolUpdating
+
+			// Fail test if realTimeKernel is already installed.
```

Comment:
May be it's better to skip test instead of failing? If rt-kernel is enabled initially - it is not an error of this test

Reply:
Done.